src/Larceny.hs

```diff
@@ -45,42 +41,43 @@ need m keys rest =
 add :: Substitution -> Template -> Template
 add mouter tpl = Template (\minner l -> runTemplate tpl (minner `M.union` mouter) l)
 
-text s = \ x y -> s
+-- works to create both Templates and Fills?
+text :: Text -> a -> Library -> Text
+text t = \_ _ -> t
```

Comment:
I wonder if this is too clever. It's neat that it works for both `Template`s and `Fill`s - where `a` is `Substitution` for the first, `Template` for the second (both ignored), but maybe it makes it harder to understand? If there were two functions, one could by `Text -> Template` and the other `Text -> Fill`, which might be easier for someone using it (even though both would be the same!)